src/util/permissions.ts

```diff
@@ -1,251 +1,260 @@
 import { cacheHandlers } from "../api/controllers/cache.ts";
-import { Guild, Role } from "../api/structures/mod.ts";
+import { Role } from "../api/structures/mod.ts";
 import { botID } from "../bot.ts";
-import { Permission, Permissions, RawOverwrite } from "../types/mod.ts";
+import { Errors, Permission, Permissions } from "../types/mod.ts";
 
-/** Checks if the member has this permission. If the member is an owner or has admin perms it will always be true. */
-export async function memberIDHasPermission(
+/** calculates the permissions this member has in the given guild */
+export async function calculateBasePermissions(
```

Comment:
```suggestion
export async function calculatePermissions(
```
Any reason as to why it is named calculate**base**Permissions()?

Reply:
Because it also checks if the userID is the ownerID. I guess calculatePermissions is fine too but can cause confusion in some cases.